### ISSUE TITLE
:w no longer works (as of NVIM v0.6.0-dev+487-gaa4f0879e)

### ISSUE THREAD

**Issue by kemiller** [asvetliakov/vscode-neovim#755]
This may have started after pulling the latest daily build of neovim (I went from a 0.5 build to a 0.6 build) but :w no longer works.  When I run :command inside VSCode, I see that the Write command is missing, but everything else is there.  A bit bizarre.  When I manually past it in from the vim sources in the plugin, it works just fine again.

**Comment by leexikang** [comment]
A fix for that pull request #725 has already merged. My temporary fix is building the extension from the source and manually installing it. Downgrading Neovim is also an option. Hope the new version will be released soon.

**Comment by kemiller** [comment]
I see the PR.  I've just edited the plugin files directly for now.

**Comment by wSedlacek** [comment]
@asvetliakov neovim has released 0.6.0,
https://github.com/neovim/neovim/releases/tag/v0.6.0

After updating I see this warning whenever I open up VSCode
```
VSCode-Neovim:

line   34:
E1208: -complete used without -nargs
line   10:
E1208: -complete used without -narg
```

#725 looks to solve the issue, when can we expect this to be released?

**Comment by asvetliakov** [comment]
Published v0.0.83
